cirq-superstaq/cirq_superstaq/job.py

```diff
@@ -192,6 +192,20 @@ def counts(self, timeout_seconds: int = 7200, polling_seconds: float = 1.0) -> D
         self._check_if_unsuccessful()
         return self._job["samples"]
 
+    def to_dict(self) -> Dict[str, Any]:
+        """Refreshes and returns job information.
+
+        Note:
+            Whenever possible, users should use the specific methods to retrieve the desired
+            information instead of this method. The contents of `_job` are likely to change pretty
+            frequently and are not guaranteed to be consistent or reliable.
```

Comment:
nit rewording
```suggestion
            The contents of this dictionary are not guaranteed to be consistent over time. Whenever
            possible, users should use the specific `SuperstaqJob` methods to retrieve the desired
            job information instead of relying on particular entries in the output of this method.
```